_includes/associates.html

```diff
@@ -5,7 +5,7 @@
     <ul class="associates n-column-list">
```

Comment:
You can get Jekyll to sort this list for you by doing:

```diff
$ git diff
diff --git a/_includes/associates.html b/_includes/associates.html
index a603edc..419c61b 100644
--- a/_includes/associates.html
+++ b/_includes/associates.html
@@ -1,4 +1,4 @@
-{% assign associates = site.data.associates.associates %}
+{% assign associates = site.data.associates.associates | sort: "name" %}
 
 <div id="associates">
 
```

Reply:
Thanks. That's a much better idea. Note that this sorts by first name, rather than last name, but I don't have any preference either way (as long as it's a stable and arbitrary ordering 🙂)

Reply:
I gave up on last name sorting years ago after starting to deal with the complexity of multiple cultures, but if you want that, you can either use separate fields for given and family names or you can just use an array, e.g. in the YAML:

```
- name: [Harding, David]
```

The sort code from my earlier comment works with that, but you also need to change the way the names are printed.  E.g.:

```diff
-            <h3 itemprop="name">{{ associate.name }}</h3>
+            <h3 itemprop="name">{% for part in associate.name reversed %}{{part}} {% endfor %}</h3>
```

Reply:
I think it's fine to order by first name, but thanks for sharing this!